edx_exams/apps/lti/tests/test_views.py

```diff
@@ -38,7 +39,11 @@ def setUp(self):
 
         self.course_exam_config = CourseExamConfigurationFactory()
         self.exam = ExamFactory()
+
+        # Create an Exam Attempt that has already been submitted.
         self.attempt = ExamAttemptFactory()
+        self.attempt.status = ExamAttemptStatus.submitted
```

Comment:
you can pass fields into ExamAttemptFactory to so it's created with status=ExamAttemptStatus.submitted instead of the default.